.circleci/generate_config_yml.py

```diff
@@ -117,14 +117,14 @@ def gen_build_workflows_tree():
             },
             "docker_build": OrderedDict(
                 {
-                    "triggers": [
-                        {
-                            "schedule": {
-                                "cron": miniutils.quote("0 15 * * 0"),
-                                "filters": {"branches": {"only": ["master"]}},
-                            }
-                        }
-                    ],
+                    #"triggers": [
+                    #    {
+                    #        "schedule": {
+                    #            "cron": miniutils.quote("0 15 * * 0"),
+                    #            "filters": {"branches": {"only": ["master"]}},
+                    #        }
+                    #    }
+                    #],
```

Comment:
@jeffdaily you probably don't want to comment that out, do you?

Reply:
The trigger will be restored once all images have successfully built.

Reply:
This should be fine to uncomment out now that the images have been built